addons/account/models/account_journal_dashboard.py

```diff
@@ -72,6 +72,26 @@ def _get_json_activity_data(self):
     show_on_dashboard = fields.Boolean(string='Show journal on dashboard', help="Whether this journal should be displayed on the dashboard or not", default=True)
     color = fields.Integer("Color Index", default=0)
     entries_count = fields.Integer(compute='_compute_entries_count')
+    has_holes = fields.Boolean(compute='_compute_has_holes')
```

Comment:
replace to `has_sequence_holes` for a better understanding

Reply:
done